app/templates/components/billing-invoices.hbs

```diff
@@ -1,22 +1,64 @@
-<section class="invoices">
-  <h2>
-    Invoice History
-  </h2>
-  <p>
-    <SvgImage @name="icon-help" @class="icon-help" />
-    Having trouble with your invoices?
-    <LinkTo @route="help" @query={{hash anchor="form"}}>
-      We’re happy to help
-    </LinkTo>
-  </p>
-  <ul>
-    {{#each (sort-by "createdAt:desc" this.invoices) as |invoice|}}
-      <li>
-        <a data-test-invoice href={{invoice.url}}>
-          <SvgImage @name="icon-invoice" @class="icon" />
-          {{invoice.id}} {{moment-format invoice.createdAt "MMMM YYYY"}}
-        </a>
-      </li>
-    {{/each}}
-  </ul>
+<section class='invoices' data-test-invoices>
+  <h2>Invoice History</h2>
+  <div class='flex flex--wrap flex--space-between'>
+    <p class='margin-top-0' data-test-help-text>
+      <SvgImage @name='icon-help' @class='icon-help' />
+      Having trouble with your invoices?
+      <LinkTo @route='help' @query={{hash anchor='form'}}>
+        We’re happy to help
+      </LinkTo>
+    </p>
+    <TravisForm as |Form|>
+      <div class='form-elem invoice-select-year'>
+        <Form.field
+          data-test-invoice-select-year
```

Comment:
I'd recommend to always add a value to attributes, even when they are boolean, e.g. 
```suggestion
          data-test-invoice-select-year="true"
```
Ember has issues with attributes without values. For instance, it will work only when it's on the first position, right next to tag name, otherwise it will fail to compile.